staging/src/k8s.io/apiserver/pkg/server/config_selfclient.go

```diff
@@ -38,12 +38,9 @@ func (s *SecureServingInfo) NewClientConfig(caCert []byte) (*restclient.Config,
 	}
 
 	return &restclient.Config{
-		// Increase QPS limits. The client is currently passed to all admission plugins,
-		// and those can be throttled in case of higher load on apiserver - see #22340 and #22422
-		// for more details. Once #22422 is fixed, we may want to remove it.
-		QPS:   50,
-		Burst: 100,
-		Host:  "https://" + net.JoinHostPort(host, port),
+		// Do not limit loopback client QPS.
+		QPS:  -1,
```

Comment:
Why?

Reply:
@answer1991 Can you explain a bit more here regarding why we shouldn't limit loopback client QPS?  Thanks!

Reply:
Sure. 

We found in our API Server lots of goruntine stuck at `limiter.Accpet()` and cost logs of time.

API Server will use this client configuration in inner controller, admission client and other client to visit API Server itself. Current configuration will limit the QPS of this client and influence the performance of API Server.

As API Server has already had `--max-mutating-requests-inflight` and `--max-requests-inflight` to prevent API Server overload from client requests, and API Server use this client in most scenes when client request is accepted. So I think we should remove this client QPS limit to improve API Server throughout capacity.

Any way, to set QPS `-1` is tricky, but there is no any other ways to remove the client QPS before we refactor lots of codes.

Reply:
Related issue: https://github.com/kubernetes/kubernetes/issues/78766